db/data/e2e/001_application_received.sql

```diff
@@ -0,0 +1,9 @@
+begin;
+
+insert into ccbc_public.application_status
```

Comment:
nit: can probably just use the 001_received_applications.sql for this

Reply:
Oh I don't think that was there when I started this PR, I will update it thanks.